Move suspend settings to pop-os/desktop

13r0ck opened the pull request:
It was decided that suspend should be enabled by default for all
users, not just systems manufactured by system76. As such, the
setting has been moved to pop-os/desktop
Commit: cb3460d394702ea22b283f3bec6923052aeea5ae

Requires: pop-os/desktop#101

jacobgkau reviewed (dismissed):
Tested with https://github.com/pop-os/desktop/pull/101.

13r0ck committed:
Remove check for manufacturer

It was decided that suspend should be enabled by default for all
users, not just systems manufactured by system76.

13r0ck commented:
It was decided by carl that systems shouldn't be radioactively changed, so this PR has been modified

jacobgkau reviewed (approved):
This no longer affects an existing user account. It still affects a new user account on an existing system.
